models/public/efficientdet-d0-tf/efficientdet-d0-tf.md

```diff
@@ -0,0 +1,82 @@
+# efficientdet-d0-tf
+
+## Use Case and High-Level Description
+
+The "efficientdet-d0" model is one of the [EfficientDet](https://arxiv.org/abs/1911.09070)
```

Comment:
>The "efficientdet-d0" model is one of the ...

Use either the name from OMZ (`efficientdet-d0-tf`) or the name from original source (EfficientDet D0)